01-davinci.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: page
+title: First Steps in LHCb
+subtitle: Introduction to the LHCb Software
+minutes: 10
+---
+
+> ## Learning Objectives {.objectives}
+> * Learn the key concepts needed to work with the LHCb software
+> * Learn how to launch the LHCb software with `lb-run`
+
+Imagine you want to design and run a new particle detector.
+Apart from organizing a collaboration, creating the design and specification, and several other tasks, you will also have to find solutions to many computational challenges.
+It's worth thinking about these for a second:
+
+ - How do we collect data as it is recorded by the detector?
+ - How do we filter and process the recorded data efficiently?
+ - How do we manage all the complex tasks required to work with collision data?
+ - How do we organize all the data of a single bunch crossing in a flexible way?
+ - How do we configure our software flexibly without having to recompile it?
+ - Can you think of more?
+
+How would you go about solving these?
+The decisions you make will affect the performance of your experiment during datataking and analysis.
+
+At LHCb, we base our software on the [Gaudi](https://proj-gaudi.web.cern.ch/proj-gaudi/) framework, which was specifically designed with the above questions in mind.
+It's worth getting an idea of some of the most important Gaudi concepts at this point.
+After this, we will jump right into running the software and getting useful things done.
+
+**Event Loop**
+Because the individual bunch crossings are almost completely independent of each other, it makes sense to process them one by one, without holding them all in memory at once.
+Gaudi provides a global *EventLoop*, which runs over individual events, and allows you to process them one by one
+
+**Transient Event Store**
+A single event contains lots of different data objects (*Particles*, *Vertices*, *Tracks*, *Hits*, ...).
+In Gaudi, these are organized in the *Transient Event Store* (TES).
+You can think of it as a per-event file system with locations like `/Event/Rec/Track/Best` or `/Event/Phys/MyParticles`.
+When running over the event stream, Gaudi allows you to `get` and `put` from/to these locations.
+
+**Algorithms**
+An *Algorithm* is a C++ class that can be inserted into the EventLoop.
+These allow you to perform a certain function for each bunch crossing (like filtering events, reconstructing particles, ...).
```

Comment:
I don't like bunch-crossing here. I think it should be very clear that the Gaudi processing is event-by-event and that Algorithms run *once* per event.